infrastructure/users_stack.py

```diff
@@ -11,21 +11,25 @@ class UsersStack(Stack):
     def __init__(self, scope: Construct, stack_id: str, **kwargs: Any) -> None:
         super().__init__(scope, stack_id, **kwargs)
 
-        account_ids = (
-            aws_iam.AccountPrincipal(account_id=account_id)
-            for account_id in os.environ.get(
-                "DATALAKE_USERS_AWS_ACCOUNTS_IDS", aws_iam.AccountRootPrincipal().account_id
-            ).split(",")
-        )
+        saml_provider_arn = os.environ.get("DATALAKE_SAML_IDENTITY_PROVIDER_ARN")
```

Comment:
I don't think this change belongs with a readme cleanup :)

Reply:
Yes it comes from #434